### PR TITLE
chore: fix whitespace-sensitive test

### LINKED ISSUE #12: inline event handling
## Type of Change

- Documentation / Website

## Summary
Want to make sure that this pattern applies (assuming it should) to help ensure declarative event handling, e.g.
```js
class Header extends HTMLElement {
  
  connectedCallback() {
    if (!this.shadowRoot) {
      this.attachShadow({ mode: 'open' });
      this.shadowRoot.innerHTML = this.render();
    }
  }

  toggle() {
    alert('this.toggle clicked!');
  }

  render() {
    return `
      <header class="header">
        <button onClick="${this.toggle()}">Button To Click</button>
      </header>
    `;
  }
}

export { Header };
customElements.define('wcc-header', Header);
```

## Details
The current example looks like this, which is more imperative.  
```js
class Header extends HTMLElement {
  
  connectedCallback() {
    if (!this.shadowRoot) {
      this.attachShadow({ mode: 'open' });
      this.shadowRoot.innerHTML = this.render();
    } else {
      const button = this.shadowRoot.querySelector('button');

      button.addEventListener('click', this.toggle);
    }
  }

  toggle() {
    alert('this.toggle clicked!');
  }

  render() {
    return `
      <header class="header">
        <button>Button To Click</button>
      </header>
    `;
  }
}

export { Header };
customElements.define('wcc-header', Header);
```

----

That said, I'm not sure if one is more or less useful for something like #11 ?  Because if we want to strip out "dead code", which one is easer to detect via AST?

### PR DESCRIPTION
**Pull request by KaiPrince**
There's another test that is sensitive to whitespace. This PR prepares it to work with the prettier formatting.

## Related Issue

#180 

## Summary of Changes

1. [x] Add `.trim` to textContent

### CODE REVIEW

File: test/cases/jsx/jsx.spec.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ describe('Run WCC For ', function() {
 
           expect(badge.getAttribute('count')).to.be.equal('0');
           expect(span.getAttribute('class')).to.be.equal('unmet');
-          expect(span.textContent).to.be.equal('0');
+          expect(span.textContent.trim()).to.be.equal('0');
```
Comment (KaiPrince) [inline test/cases/jsx/jsx.spec.js]: I could have done a replace on whitespace, but I figured this was simpler for now.

**Comment by thescientist13** [comment]
Oh yeah, I see the `trim()` usage now.  I am fine with that, and if so, I think #180 would be good to go from my end then.  👍

**Comment by KaiPrince** [comment]
Ah okay yeah I snuck it in there but I thought I should create this PR separately to highlight the change, lest it escape notice in that 100+ file PR
